internal/http_api/topic_channel_args.go

```diff
@@ -10,24 +10,40 @@ type getter interface {
 	Get(key string) (string, error)
 }
 
-func GetTopicChannelArgs(rp getter) (string, string, error) {
+// GetChannelArg returns the ?topic parameter
```

Comment:
minor typo: s/Channel/Topic/

Reply:
If that's all i get from your review i'll take it! 😉